docs/pages/server-access/guides/bpf-session-recording.mdx

```diff
@@ -57,19 +48,79 @@ $ uname -r
 </tbody>
 </table>
 
-## Step 1/3. Install and configure Teleport node
+(!docs/pages/includes/tctl.mdx!)
+
+## Background
+
+Teleport Nodes submit SSH session traffic to the Auth Service for
+storage, copying what is echoed in a terminal. These recorded sessions can be
+replayed later via the `tsh play` command or in the Teleport Web UI.
+
+The advantage of this approach is that, since no input is captured, these
+session recordings do not contain passwords that were entered into a terminal.
+
+The disadvantage is that session recordings can be bypassed using several
+techniques:
```

Comment:
Maybe reword this? The sessions are still recorded, so I'm not sure I would describe it as session recording being bypassed.

The disadvantage is really that the users true actions can be hidden or obfuscated in such a way that it's hard to determine what actually happened by watching the recording (even though the recording does accurately reflect the session).